RELEASE.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+# Release process automation script. 
+# Used to create branch/tag, update VERSION files and and trigger release by force pushing changes to the release branch 
+
+# set to 1 to actually trigger changes in the release branch
+TRIGGER_RELEASE=0 
+
+#REPO=git@github.com:eclipse/che-machine-exec
+#VERSION=7.7.0
```

Comment:
Leftover notes? I hope we're not planning on updating this to 7.8.0 in 3 weeks :)

Reply:
those be comments, yo. sample values. But yeah I can remove that

Reply:
Fair enough